java/arcs/core/policy/PolicyOptions.kt

```diff
@@ -10,12 +10,5 @@ data class PolicyOptions(
      *
      * Temporary measure until we have a proper way to designate these stores.
      */
-    val storeMap: Map<StoreId, String>,
-
-    /**
-     * Maps policy with the allowed egress particles.
-     *
-     * Temporary measure until we have a proper way to designate egress particles.
-     */
-    val policyEgresses: Map<String, List<String>>
+    val storeMap: Map<StoreId, String>
```

Comment:
Are we not deleting `PolicyOptions`?

Reply:
We still need the storeMap for now, unfortunately. Unless you have a suggestion for how to get rid of it too?

Reply:
Don't have a good way to avoid it right now. SG.